docs/public/svg/integrations/flutter.svg

```diff
@@ -1,9 +1,11 @@
-<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 64 64" width="64" height="64">
-	<defs>
-		<image width="46" height="56" id="img1" href="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAC4AAAA4CAMAAAC16DqDAAAAAXNSR0IB2cksfwAAAuJQTFRFAAAARtT+RdL9RdH9RdL9RdP+RdL9RdP9RdH9RtT+RdL9RdL9RtP+Rtb9RdL9RdP+RdD8RtT9RdL9RdL9RdT+RND9RtX9RdL9RdL9RdT+RtT+RdL9RdL9RtT+RtT+RdL9RdP+RtD9RtT+RtP+R9L+RtT+RtT+RdL9RtX+Rs/9RtT9RtL+RtP+RdL9RdD9RtP+RtH+RtT+RdH9RdH+RtP+RdL9RNH8RdP9RdH9RdL9RdH9RdL9Rtb+RdL9RdH9RdL9RdT+RdL9RdH9RtX+RdL9RdH9RNH9RdL+RdH9Rtb9RdL+RdL9RdT9RdH+RdH9RtX9RdH8RdD9RdL+RdH9RdP9RdL+RdH9RdL9Rtb+RNH9RNH+RNH+RdH9RdL+RdH9RtX+RNH+RNH+RNH+RNH+RNH9RtL9RdH9RdT9Q9H+RNH+RNH+RNH+RNH+RdP9Rtb+RNH+RNH+Q9H+Q9H9RNH+RNH+RdP9RdL9RNH+RNH+RdH/RdP9RdP9RNH+RNH+Q9D9PMLzQs37RNH+RNH9RdP9RdH9RdP9RtT+PMLyOr/wO8DwQcv5RNH+RdL9RdH9RdL9RtX+PMLyOr/wOr7vO8DxQs78RNH+Q9L+RdP9RdL9RdL9Rtf+Qcz6PMLyOr/wRNH+RND+RdP+Rtb+PcLyO7/wQs37Q9D+PMT0O7/wPMT0Or/wOr/wQs37RNH+PcTzO7/wPsf2O7/wPMPzRND9P8j3O8DwMa3iDGapAlSdPsb1O8DwKZ3VCF+jAlOaAlOaAlScAlOaPcT0Or/wMa3hAlObAVKZPcX0Or/wK6LYCmOnAlOaPcX0O8DxLabbBVmfAlOaAlWdPcX0O7/wAlScAlOaPcX1O8DxK6HYAlScAlOZOr/wMKrfAlOaAlScKqHZBluhAlOaAlScAVOZAlScAlObAlObAlScAlSbAlOaAlScAVKZAlSbAlOaAlScAlWcAlObAlKZAlSdAlSbAlScAlWdAlObAlOaYQL8JQAAAPZ0Uk5TADPr/9orzEYANOHkKzDXLwA17NkrAEDi1io23eMuPNMqADcyACpO7RYAOABZwQA6AFHuADvvAEjlU/fUDkTwwClf9Bg+5wBq8Q5G0Sli9RsAAVn4023yvxQTHQwBZPkXJrvKqRli6SgmqNm4OD0NL8IaHrE3R97BIgAnviHAGjXGtxom3bEXLuP747JH3r4aLNf/+d4oACPStw0BNfy0GTMTNc7fJivjPuH94Bo14jr8tBMy4//eLynW////80QBP9T/2gAk3P//6iPI///0SB/MRQAcv/9UAdH/6EYXsfxOARS77xK/60cDqvVhEbgBEZZsEcDsOAbakAAAAmVJREFUeJyV0mV001AcBfBAkcGQwShabFCcAcMGQ4YNd3cZDsWl6Bg6HF5xd3eHPorDBqy4F7fBcPlOu6Rtctv3P4f7KWl+557bl0iSOKlS67RJk5bS6VCnDxDrDBlBB2YidGZckiWrGAdlyw46OIe4W58zF+jceYglefOBNuQnugvgkoKFxNWFQwJxdxG9UBcNMeASI7G7WHH8lyXEuGQp1MFGsS5dBneXNYaKeTnUBkKXrxCG3cTuipUq/8fuKlXDfXYTJ1itOuiIGmKsr1kLdO1IQtepi7oe8VXVb4BLIqOEOrRhI+xuTCxpgrppM7EOat4Cu1u2EurWbdqCbkcsiWrfAXTHTsqjzl0gXaVu3Xug7qnoXr37aBPdV+rXH/UARQ8cNFibIUMDJFME6GGKHj4C9UjXz6Ywv92jRqMeIz8YO87f7vGozROURxM9H+4k95LJqKdM9Z6lKRx2x0wDHTtdffYzZrr0rNnK7Zy4ufM0er5Z+64WLNTpFi1WbpYsZcyyTN29fAW825WrVq+Rr9aui2POrPf2b9jo+y1s2qxcbNnKUmLZ5u7e7qs92bGTKbHI/bFmQu/azTzZs9epo/cRev8Bpoqz399uTw5qNGOHDh8htHT0mJYfP3GS4tKp02p9xsrP2s5R/vwFr75o5Zxfsl2m/JWrbn0tnruSYCP3XL8h65uJXI79Fulv30nRVu6O/e49yt9/wNjDRO5NwqPHpH/yNJ6r88xB7nn+4qWGc/ur15R/81bL+TsHeZ7vP4D/mET2f/oMPvkLuf/rN9ifnPSd8j9+4p5f5Hn+/gP+r+MfAcCTYdekpV0AAAAASUVORK5CYII="/>
-	</defs>
-	<style>
-		tspan { white-space:pre }
-	</style>
-	<use id="Layer 1" href="#img1" transform="matrix(0.978,0,0,0.978,10,5)"/>
-</svg>
\ No newline at end of file
+<svg width="44" height="44" viewBox="0 0 44 44" fill="none" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
```

Comment:
Looks like the `flutter` svg is not aligned with the other three. We should probably adjust `height`, `width` and `viewBox` there

![Screen Shot 2022-02-24 at 3 15 34 PM](https://user-images.githubusercontent.com/40295569/155623576-238fd441-b536-44ba-8c11-b571ed8f5274.png)

Reply:
I played around with height, width and viewBox and couldn't get it right. I ended up undoing most of those changes and just added some padding to make it look right (although I am sure it still isn't perfect).

The react/vue/angular svgs don't occupy the full view box so there is essentially padding built in (not sure of a better term for that). I tried to add equal padding to the flutter logo.

I will note that even the react/vue/angular svgs are not the exact same size. The react logo sticks up further than both of the others. The Angular logo is the smallest, which might be part of the reason the flutter logo looks so big.

I can try to play around with it a little bit more if you would like. Are you thinking the flutter logo has too much height?

Reply:
I pushed updated. Here is how it looks now.

![Screen Shot 2022-02-24 at 6 30 55 PM](https://user-images.githubusercontent.com/20613561/155624967-34f5eab8-b65b-4cd4-b9bf-741ca27a4744.png)

Reply:
I know ahhhh. It is painful to adjust these props :)